core/templates/dev/head/base.html

```diff
@@ -225,10 +225,44 @@
                             </ul>
                           </li>
                         {% endif %}
+                        {% if nav_mode != 'create' and nav_mode != 'explore' %}
+                          <create-exploration-button ng-if-start="windowWidth > 768"></create-exploration-button>
```

Comment:
Yes, my apologies. (Though, in that case, the answer to the question "Can we do it just once for the entire code branch?" should probably have been "No, because ...".)

I assume the styling issues are due to bootstrap and not oppia.css?